_posts/2016-12-04-contributing-blog-post-to-pulpproject.org.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Contributing a blog post to pulpproject.org
+layout: post
+author: Dennis Kliban
+---
+Quickstart Guide
+----------------
+
+1. Fork the pulpproject.org repository on GitHub.
+   [https://github.com/pulp/pulpproject.org/](https://github.com/pulp/pulpproject.org/)
+
+1. Clone your fork locally and checkout the `gh-pages` branch.
+
+1. Create a new file in the `_posts` directory. The name for the file should be in the following
+   format: `2016-11-30-contributing-blog-post-to-pulpproject.org.md`
```

Comment:
Can a generic example be given showing the filename format. Then this concrete example would make more sense.